code/client/src/sdk/mafia/ui/hud/c_hud_controller.h

```diff
@@ -1,9 +1,44 @@
 #pragma once
 
+#include <utils/hooking/hooking.h>
+
 namespace SDK {
     namespace mafia::ui::hud {
+
+        class C_RaceTimer {
```

Comment:
keep or remove? might be useful for future reference.

Reply:
keep